org.widelands.widelands.json

```diff
@@ -0,0 +1,478 @@
+{
+    "app-id": "org.widelands.widelands",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "widelands",
+    "rename-icon": "widelands",
+    "rename-desktop-file": "widelands.desktop",
+    "rename-appdata-file": "widelands.appdata.xml",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--persist=.widelands",
+        "--device=dri"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g -Wno-deprecated-declarations -fstack-protector-strong -D_FORTIFY_SOURCE=2",
+        "cxxflags": "-O2 -g -Wno-deprecated-declarations -fstack-protector-strong -D_FORTIFY_SOURCE=2",
+        "ldflags": "-fstack-protector-strong -Wl,-z,relro,-z,now",
+        "env": {
+            "V": "1"
+        }
+    },
+    "modules": [
+        {
+            "name": "xmlstarlet",
+            "config-opts": [
+                "--disable-static-libs",
+                "--with-libxml-libs-prefix=/usr/lib",
+                "--with-libxml-include-prefix=/usr/include/libxml2",
+                "--prefix=/app/opt/xmlstarlet",
+                "--exec-prefix=/app/opt/xmlstarlet"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
```

Comment:
Do you actually need it after build?

Reply:
As you can see, at the end I remove everything from `/opt`:
https://github.com/scx/flathub/blob/c36ced8dea9b50e1452e578a3fb3806ff005e9a5/org.widelands.widelands.json#L474

Reply:
You should use `cleanup` instead.

Reply:
And what is wrong with the current `cleanup`?
https://github.com/scx/flathub/blob/c36ced8dea9b50e1452e578a3fb3806ff005e9a5/org.widelands.widelands.json#L466

Reply:
Missed that. I think per module `"cleanup": [*]"` is something more "idiomatic", but that's fine too.